### PR TITLE
Fix deprecated api for map

### PR DESCRIPTION
**Pull request by ananya**
Fixes : Fix deprecated api for map.
in progress.
Changes: Added api for getting user ip for latitude and longitude information

Demo Link: https://pr-1919-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by ananya-agrawal** [comment]
@AakashMallik done

File: src/apis/index.js [hunk 1]
```diff
@@ -138,8 +139,15 @@ export function postSkillFeedback(payload) {
...
+  return ajax.get(`${USER_IP_URL}`, {
+    token: 'b3398b832bf0d7',
+  });
+}
+
 export function getDefaultMapData() {
-  return ajax.get('http://api.ipstack.com/134.201.250.155', {
-    access_key: '31fe38d0ec112087bc85f47241a58387',
+  const ip_user = getUserIP();
+  return ajax.get('http://api.ipstack.com/' + ip_user, {
```
Comment (AakashMallik) [inline src/apis/index.js]: use template string

File: src/components/ChatApp/MessageListItem/MessageListItem.react.js [hunk 2]
```diff
@@ -86,6 +86,25 @@ class MessageListItem extends React.Component {
       this.onYouTubePlayerReady,
     );
   }
+
+  render() {
```
Comment (AakashMallik) [inline src/components/ChatApp/MessageListItem/MessageListItem.react.js]: is it just me or there are in fact 2 render functions in the same component. Did you run the code before submitting?

**Comment by fragm3** [comment]
@ananya-agrawal please link PR to issue number

**Comment by ananya-agrawal** [comment]
Fixes #1930

File: src/apis/index.js [hunk 3]
```diff
@@ -138,8 +140,15 @@ export function postSkillFeedback(payload) {
   });
 }
 
+export function getUserIP() {
+  return ajax.get(`${USER_IP_URL}`, {
+    token: 'b3398b832bf0d7',
```
Comment (samagragupta) [inline src/apis/index.js]: Please use double quotes.

Comment (AakashMallik) [reply src/apis/index.js]: @samagragupta The linter apparently changes it back to single quotes.

File: src/apis/index.js [hunk 4]
```diff
@@ -138,8 +140,15 @@ export function postSkillFeedback(payload) {
...
+  });
+}
+
 export function getDefaultMapData() {
-  return ajax.get('http://api.ipstack.com/134.201.250.155', {
-    access_key: '31fe38d0ec112087bc85f47241a58387',
+  const ip_user = getUserIP();
```
Comment (AakashMallik) [inline src/apis/index.js]: This is not supposed to work. getUserIP() returns a promise and not an actual ip address. Did you test this code?

**Comment by akshatnitd** [comment]
Closing due to inactivity.

**Pull request closed without merging**